hardware/arduino/sam/cores/arduino/USB/CDC.cpp

```diff
@@ -196,12 +196,13 @@ void Serial_::accept(void)
 			udd_ack_fifocon(CDC_RX);
 			break;
 		}
-		c = USBD_Recv(CDC_RX);
-		// c = UDD_Recv8(CDC_RX & 0xF);
-		buffer->buffer[buffer->head] = c;
-		buffer->head = i;
-
-		i = (i + 1) % CDC_SERIAL_BUFFER_SIZE;
+		uint8_t c[CDC_SERIAL_BUFFER_SIZE];
```

Comment:
No, array is used instead of variable now. Pls check carefully and pls read my decription above